mix.exs

```diff
@@ -50,10 +51,11 @@ defmodule Membrane.Element.UDP.MixProject do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.8.0"},
+      {:membrane_core, "~> 0.9.0"},
       {:ex_doc, "~> 0.21", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 1.0.0", only: [:dev, :test], runtime: false},
-      {:mockery, "~> 2.3.0", runtime: false}
+      {:mockery, "~> 2.3.0", runtime: false},
```

Comment:
Nitpick: can you group together prod, dev-only and test-only deps? I.e. move mockery just below core